helm-util/README.md

```diff
@@ -0,0 +1,50 @@
+# helm-util
+
+Provides utilities for using helm in jsonnet.
+
+## Helmraiser
+
+Helmraiser refers to Go package that provides helmTemplate native function through Tanka for generating a Jsonnet
+library from any Helm chart. The goal of this function is to keep track of upstream changes in a Helm chart while
+retaining Jsonnet changes (read: mixins) on top of them.
+
+## Example usage
+
+```
+jb install github.com/grafana/jsonnet-libs/helm-util
+```
+
+```
+local helm = import 'github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet';
+
+{
+    version:: 'v0.16.0',
+
+    values:: {
+      installCRDs: true,
+      global: {
+        podSecurityPolicy: {
+          enabled: true,
+          useAppArmor: false,
+        },
+      },
+    },
+
+    labels:: {
+      'custom_label': 'example',
+    }
+
+    chart: helm.helmTemplate(
+      name='cert-manager',
+      chart='jetstack/cert-manager',
+      $.conf,
+      conf={
+        values: $.values,
+        flags: [
```

Comment:
Let's instead accept an Object here and use the `mapToFlags` code (copy from `kausal.libsonnet`) to convert to flags.

Flags are an implementation detail after all and if we ever move off them (e.g. vendoring Helm code instead of shelling out) it's far easier working with objects than having to translate flags into variables

Also doesn't need string templating :)

Reply:
As mentioned earlier, I don't want to implement it now, having the ability to play with the raw flags at this stage of development is far more valuable to me than abstracting it away.

Reply:
We are creating an API here, which should stay as stable as possible.

Having those flags here is not acceptable for a good API IMO. Furthermore, `mapToFlags` does not abstract things away, it's a more generic way to express the same. There is no functional difference, you still have access to all flags you might want

Reply:
To not hold this any longer, let's go ahead with this version and deprecate it before releasing Tanka 1.0

Reply:
I'm 100% sure flags/command line arguments can be represented as an array of strings (for reference: the k8s container spec, this is also represented args like this). If we want to do more specific things and use a map in future releases, we can either parse the flag strings or introduces a new map that holds specific options as part of the 'conf' map argument in the native command, it should leave us enough room for growth.